docs/development/testing_and_dependencies.md

```diff
@@ -1,7 +1,9 @@
 # Testing
+
 We follow the BDD-style testing principles and are leveraging the [Ginkgo](https://onsi.github.io/ginkgo/) framework along with [Gomega](http://onsi.github.io/gomega/) as matcher library. In order to execute the existing tests, you can use
 
 ```bash
+$ make verify
```

Comment:
You could leave only `make verify` or `make test`, or better - explain the diff between them. Maybe something like:
```suggestion
$ make tests        # runs tests
$ make verify       # runs static code checks and test 
```